src/namespace.js

```diff
@@ -178,7 +178,7 @@ class Namespace {
    */
   get elements() {
     if (this._elements === undefined) {
-      var name, pascal;
+      var name; var pascal;
```

Comment:
Shouldn't these be two lines?

Reply:
Yes, but that comes in later rules.

In later commit the whole line goes away since we're using forEach so name vanishes and made pascal a const inside the forEach block.